library/modules/bigip_profile_client_ssl.py

```diff
@@ -330,6 +480,21 @@ def cert_key_chain(self):
         result = self._diff_complex_items(self.want.cert_key_chain, self.have.cert_key_chain)
         return result
 
+    @property
+    def options(self):
+        if self.want.options is None:
+            return None
+        if self.have.options is None and self.want.options == '':
+            return None
+        if self.have.options is not None and self.want.options == '':
+            return []
+        if self.have.options is None:
+            return self.want.options
+        if set(self.want.options) == set(self.have.options):
```

Comment:
this condition here can be removed. Its sufficient to check the `!=` condition only because at this point we've checked the "side cases" (what you have above) and all that is left is to check the not-equal case because, by default, if nothing is returned, then nothing will be reported.